plugins/woocommerce/src/Internal/Admin/Settings/PaymentProviders/WooPayments/WooPaymentsService.php

```diff
@@ -247,6 +248,20 @@ private function get_onboarding_steps_details( string $location, string $rest_pa
 
 		$details[] = $business_verification_step_details;
 
+		// Add the finish onboarding step details.
+		$finish_step_details = array(
```

Comment:
The finish step is not a back-end step, so we shouldn't include it here. Unless we mark the site with some tag, but I don't think it's happening here.

Reply:
Since the finish step requires the overview page to be provided by the backend, I thought it made sense to handle it as a backend step. Otherwise, we’d need to modify the OnboardingContext to pass data from a previous step to the final one, which isn’t an ideal approach. WDYT?

Reply:
@elazzabi, as we discussed on Slack, we’ve updated how the OnboardingContext handles data, so this step no longer needs to be managed on the backend.